website/src/app/kb/user-guides/linux-client/readme.mdx

```diff
@@ -56,18 +57,27 @@ A sample output of the help command is shown below:
 ```
 > sudo ./linux-client-x64 -h
 
-Usage: linux-client-x64 [OPTIONS] --firezone-id <FIREZONE_ID> <TOKEN> [MAX_PARTITION_TIME]
+Usage: linux-client-x64 [OPTIONS] [COMMAND]
 
-Arguments:
-  <TOKEN>               Token generated by the portal to authorize websocket connection [env: FIREZONE_TOKEN=]
-  [MAX_PARTITION_TIME]  Maximum length of time to retry connecting to the portal if we're having internet issues or it's down [env: MAX_PARTITION_TIME=] [default: 30d]
+Commands:
+  auto        If there is a token on disk, run in standalone mode. Otherwise, run as an IPC daemon. This will be removed in a future version
+  daemon      Listen for IPC connections and act as a privileged tunnel process for a GUI client
+  standalone  Act as a CLI-only Client, don't listen for IPC connections
+  help        Print this message or the help of the given subcommand(s)
 
 Options:
-  -n, --firezone-name <FIREZONE_NAME>  Friendly name to display in the UI [env: FIREZONE_NAME=]
-  -i, --firezone-id <FIREZONE_ID>      Identifier generated by the portal to identify and display the device [env: FIREZONE_ID=]
-  -l, --log-dir <LOG_DIR>              File logging directory. Should be a path that's writeable by the current user [env: LOG_DIR=]
-  -h, --help                           Print help
-  -V, --version                        Print version
+      --token <TOKEN>
+          Token generated by the portal to authorize websocket connection [env: FIREZONE_TOKEN=]
+  -i, --firezone-id <FIREZONE_ID>
+          Identifier used by the portal to identify and display the device [env: FIREZONE_ID=]
+  -l, --log-dir <LOG_DIR>
+          File logging directory. Should be a path that's writeable by the current user [env: LOG_DIR=]
+  -m, --max-partition-time <MAX_PARTITION_TIME>
+          Maximum length of time to retry connecting to the portal if we're having internet issues or it's down. Accepts human times. e.g. "5m" or "1h" or "30d" [env: MAX_PARTITION_TIME=]
+  -h, --help
+          Print help
+  -V, --version
+          Print version
```

Comment:
Pasted the new help text.

Should I hide the IPC daemon bit, or...?

Reply:
haha well it'll be hidden in the systemd file or something. 

I hid all those commands so it just has `standalone` and `help`. The KB example with no subcommand is equivalent to the hidden subcommand `auto` which auto-detects whether to go to IPC service mode (formerly 'daemon') or standalone.